src/controls/listPicker/ListPicker.tsx

```diff
@@ -1,13 +1,13 @@
-import * as React from 'react';
```

Comment:
@russgove for future PRs, if possible please avoid re-ording imports. I assume this was done by a VS code extension?